mobile-widgets/qml/main.qml

```diff
@@ -13,7 +13,7 @@ Kirigami.ApplicationWindow {
 	id: rootItem
 	title: qsTr("Subsurface-mobile")
 	reachableModeEnabled: false // while it's a good idea, it seems to confuse more than help
-	wideScreen: true // workaround for probably Kirigami bug. See commit.
+	wideScreen: false // workaround for probably Kirigami bug. See commits.
```

Comment:
I think we should discuss this with @notmart 
But I'll take the commit for now.